templates/geosafe/analysis/create.html

```diff
@@ -976,6 +979,38 @@
             });
         }
 
+        function set_transparency() {
+            var exposure_id = $(".exposure.section a.selected").attr('data-id');
+            var hazard_id = $(".hazard.section a.selected").attr('data-id');
+            //make sure the exposure or hazard layers has selected.
+            if (exposure_id == undefined && hazard_id == undefined)
+            {
+                return;
+            }
+            exposure_layer = add_layer(map, layer_tiles_url, exposure_layer);
+            hazard_layer = add_layer(map, layer_tiles_url, hazard_layer);
+            $('#transparency-option').popover({
+                    html: true,
+                    placement: 'right',
+                    title: 'Set Transparency',
+                    content: function () {
+                        return $('#popover-transparency').html();
+                    }
+                }).click(function () {
+                    //Using slider from https://github.com/seiyria/bootstrap-slider
+                    //there are two slider: jquery and seiyria's version
+                    //calling slider with bootstrapSlider to distinguish between them according to: http://stackoverflow.com/questions/26691018/using-bootstrap-slider-with-jquery-ui
+                    $('#rangeSlider').bootstrapSlider().on('slide', function (ev) {
```

Comment:
Use the same convention with geonode for id and class name.
Change #rangeSlider to #range-slider